client/root.js

```diff
@@ -99,6 +112,10 @@ async function getLoggedInLandingPage( { dispatch, getState } ) {
 	const isCustomerHomeEnabled = canCurrentUserUseCustomerHome( getState(), primarySiteId );
 
 	if ( isCustomerHomeEnabled ) {
+		if ( isClassicAdmin ) {
+			// This URL starts with 'https://' because it's the access to wp-admin.
+			return `${ primarySiteURL }/wp-admin/`;
```

Comment:
I think we can just use the existing `getSiteAdminUrl()` selector here, which is more "robust"

Reply:
Thanks